openbas-front/src/admin/components/common/injects/ImportUploaderInjectFromXlsInjects.tsx

```diff
@@ -109,8 +124,26 @@ const ImportUploaderInjectFromXlsInjects: FunctionComponent<Props> = ({
     handleSubmitForm(onSubmitImportInjects)(e);
   };
 
+  const checkNeedLaunchDate = () => {
+    const formValues = getValues();
+    if (formValues.importMapperId && formValues.sheetName && formValues.timezone) {
+      const input: InjectsImportInput = {
+        import_mapper_id: formValues.importMapperId,
+        sheet_name: formValues.sheetName,
+        timezone_offset: moment.tz(formValues.timezone).utcOffset(),
+      };
+      injectContext.onDryImportInjectFromXls?.(importId, input).then((value: ImportTestSummary) => {
+        const criticalMessages = value.import_message?.filter((importMessage: ImportMessage) => importMessage.message_level === 'CRITICAL');
+        if (criticalMessages && criticalMessages?.filter((message) => { return message.message_code === 'ABSOLUTE_TIME_WITHOUT_START_DATE'; }).length > 0) {
+          setNeedLaunchDate(true);
+        }
+      });
+      setNeedLaunchDate(false);
+    }
```

Comment:
This is okay, but I'd prefer to add a catch block or use await and include the conditionals for setNeedLaunchDate(false). You could move also setNeedLaunchDate(false) to the beginning of the function to show that it is the default value (it s just for better readability).